python-modules.sh

```diff
@@ -96,7 +97,7 @@ fi
 MATPLOTLIB_TAG="3.0.3"
 if [[ $ARCHITECTURE != slc* ]]; then
   # Simply get it via pip in most cases
-  env PYTHONUSERBASE=$INSTALLROOT pip3 install --user "matplotlib==$MATPLOTLIB_TAG"
```

Comment:
Why did you remove this? I guess this is the culprit...

Reply:
Looks like it could  be problem ()
In my first builds (from virtualenv described in JIRA), I had to disable this --user flag.
Testing the code  again, reverting back --user, the 
````
aliBuild build Python-modules  --defaults o2 --debug
````
is working.
````
==> Build of Python-modules successfully completed on `miranov-Strix-17-GL703GE'.
    Your software installation is at:
      /home2/miranov/software/alicesw3/sw/ubuntu1804_x86-64
    You can use this package by loading the environment:
      alienv enter Python-modules/latest-o2
````
Now i'm waiting for full build